daq/gateway.py

```diff
@@ -61,6 +61,7 @@ def _initialize(self):
         host = self.runner.add_host(host_name, port=host_port, cls=cls, tmpdir=self.tmpdir)
         host.activate()
         self.host = host
+        self.host.cmd('./change_leasetime %s' % self.runner.config.get("initial_dhcp_lease_time"))
```

Comment:
i'm torn if this should have units specificed, e.g., initial_dhcp_lease_sec so the value would be 500 vs. 500s.  Something about having "500s" seems... non-standard (I think it makes people warm and fuzzy to have time configuration parameters be integers). whatever, just something to consider. Not gonna stop anything either way.

But, alternatively, pick either _lease_time or _leasetime and be consistent :-)

Reply:
I didn't force the lease time to be seconds so any unit that dnsmasq supports can be passed in. It's also convenient to change to seconds if necessary in the future.